play2earn/mint/typechain-types/@openzeppelin/contracts/access/Ownable.ts

```diff
@@ -0,0 +1,176 @@
+/* Autogenerated file. Do not edit manually. */
```

Comment:
should this be put in the source? NPM modules should usually be put in ignore

Reply:
this should not be included, i'll remove it